internal/codegen/values.go

```diff
@@ -60,6 +60,9 @@ type git struct {
 type config struct {
 	// Name is the name of this repository
 	Name string
+
+	// Versions are versions of applications to be used over baked in defaults
+	Versions map[string]string
```

Comment:
Also curious of usage here as well.

Reply:
This is kinda a placeholder for stencil-golang, will think about it more as I work on it